pkg/cmd/registry/artifact/crud/create/create.go

```diff
@@ -186,4 +246,37 @@ func runCreate(opts *options) error {
 	}
 
 	return dump.Formatted(opts.IO.Out, opts.outputFormat, metadata)
+
+}
+
+func loadReferences(dataAPI *registryinstanceclient.APIClient, opts *options) ([]registryinstanceclient.ArtifactReference, error) {
```

Comment:
Is there any reason for someone to have a '=' in the reference name or ':' in the groupname or version?